docs/reference/migration/migrate_8_0/index-setting-changes.asciidoc

```diff
@@ -23,7 +23,7 @@ Discontinue use of the `index.max_adjacency_matrix_filters` index setting.
 
 Requests that include the index setting will return an error. If you upgrade a
 cluster with a 7.x index that already contains the setting, {es} will
-{ref}/archived-settings.html#archived-index-settings[archive the setting].
+{ref}/setup-upgrade.html#archived-settings[archive the setting].
```

Comment:
There is no archived setting on this page.

Should the link go to this page: https://www.elastic.co/guide/en/elasticsearch/reference/master/archived-settings.html